packages/docs/PropTables/StatefulTablePropTable.tsx

```diff
@@ -19,6 +19,11 @@ const statefulTableProps: Prop[] = [
     description: 'The array of items to display.',
     required: true,
   },
+  {
+    name: 'dragAndDrop',
```

Comment:
🍹 Just want to have a discussion about this prop. Should this potentially be something different? Maybe just `draggable`?